coalib/processes/Processing.py

```diff
@@ -437,13 +437,18 @@ def yield_ignore_ranges(file_dict):
         stop_ignoring = False
         for line_number, line in enumerate(file, start=1):
             # Before lowering all lines ever read, first look for the biggest
-            # common substring, case sensitive: I*gnor*e, start i*gnor*ing.
-            if 'gnor' in line:
+            # common substring, case sensitive: I*gnor*e, start i*gnor*ing
+            # and for N*oqa start n*oqa
```

Comment:
I don't think anyone uses `start noqa`.

Also the `*`s in the `I*gnor*e` signify the substring you're searching for. You start a `*` with `N*oqa`, but it's never ended.

So this should just be `case sensitive: I*gnor*e, start i*gnor*ing, N*oqa*.`